frontend/src/features/env/SwitchEnvFeedbackModal.tsx

```diff
@@ -147,82 +146,115 @@ export const SwitchEnvFeedbackModal = ({
         ) : (
           <chakra.form noValidate onSubmit={handleFormSubmit}>
             <ModalHeader pr="48px">
-              Something not right on the new FormSG?
+              {user
+                ? 'Something not right on the new FormSG?'
+                : "Can't submit this form?"}
             </ModalHeader>
             <ModalBody mt="0" pt="0">
               <Stack spacing="1rem">
                 <FormControl>
                   <Input type="hidden" {...register('url')} value={url} />
                 </FormControl>
-                <FormControl
-                  isRequired
-                  isInvalid={!isEmpty(errors) || !!othersInputError}
-                >
-                  <FormLabel>
-                    Why are you switching to the previous FormSG?
-                  </FormLabel>
-                  <Radio.RadioGroup>
-                    {COMMON_RADIO_OPTIONS.map((option) => (
-                      <Radio
-                        {...register('switchReason', {
-                          required: {
-                            value: true,
-                            message: 'This field is required',
-                          },
-                          deps: [FEEDBACK_OTHERS_INPUT_NAME],
-                        })}
-                        value={'I’m not used to the new FormSG'}
-                        key={option}
-                        tabIndex={1}
-                      >
-                        {option}
-                      </Radio>
-                    ))}
-                    {radioOptions.map((option) => (
-                      <Radio
+                {user && isAdminView ? (
+                  <FormControl
+                    isRequired={isAdminView}
+                    isInvalid={
+                      isAdminView && (!isEmpty(errors) || !!othersInputError)
+                    }
+                  >
+                    <FormLabel>
+                      Why are you switching to the previous FormSG?
+                    </FormLabel>
+                    <Radio.RadioGroup>
+                      {COMMON_RADIO_OPTIONS.map((option) => (
+                        <Radio
+                          {...register('switchReason', {
+                            required: {
+                              value: true,
+                              message: 'This field is required',
+                            },
+                            deps: [FEEDBACK_OTHERS_INPUT_NAME],
+                          })}
+                          value={'I’m not used to the new FormSG'}
```

Comment:
Is that correct? Why is the value not coming from the `option` object?

Or alternatively, if there is only one option by design, why is `COMMON_RADIO_OPTIONS` an array?

Reply:
oh yeah, good spot! changed this to be `value={option}` ([007eab9](https://github.com/opengovsg/FormSG/pull/5339/commits/007eab92828cca687738b5e0d0abcbe9d1e4658a))

I initially created `COMMON_RADIO_OPTIONS` as an array so we'd have the flexibility to add other options easily, since we seem to be making frequent updates to the feedback modal